Ports/stress-ng/patches/ReadMe.md

```diff
@@ -0,0 +1,58 @@
+# Patches for stress-ng on SerenityOS
+
+## `0001-serenity-Disable-linux-scheduler-integration-on-Sere.patch`
+
+Subject: serenity: Disable linux scheduler integration on Serenity
+Follow the path of other platforms, and make this code nop
+when compiling for serenity.
+
+## `0002-serenity-Disable-itimer-testing-when-compiling-for-S.patch`
+
+Subject: serenity: Disable itimer testing when compiling for Serenity
+The itimer APIs are not implemented in serenity, so just disable
+these tests.
+
+## `0003-serenity-Fix-duplicate-definition-of-ALWAYS_INLINE-o.patch`
+
+Subject: serenity: Fix duplicate definition of ALWAYS_INLINE on serenity
+
+## `0004-serenity-ifdef-out-key_t-type-usage-which-serenity-d.patch`
+
+Subject: serenity: ifdef out key_t type usage, which serenity does not have
+
+## `0005-serenity-Mark-hsearch-stressor-as-not-implemented-on.patch`
+
+Subject: serenity: Mark hsearch stressor as not implemented on Serenity
+
+## `0006-serenity-Disable-signal-code-validation-for-Serenity.patch`
+
+Subject: serenity: Disable signal code validation for Serenity
+We don't currently have a definition for SEGV_ACCERR, so
+this validation is meaningless for SerenityOS.
+
+## `0007-serenity-Disable-rand48-cpu-stressor-it-s-not-implem.patch`
+
+Subject: serenity: Disable rand48 cpu stressor, it's not implemented for Serenity
+
+## `0008-serenity-Make-lsearch-stressor-a-nop-on-Serenity.patch`
+
+Subject: serenity: Make lsearch stressor a nop on Serenity
+
+## `0009-serenity-Fake-O_SYNC-for-serenity-so-iomix-stressor-.patch`
+
+Subject: serenity: Fake O_SYNC for serenity so iomix stressor compiles
```

Comment:
It looks like the other generated `ReadMe.md`'s don't include the `Subject:` prefix, but this one does for some reason. Do you know what's up with that?

Reply:
oh I guess the regex is wrong?
I'll take a look :thonk:

Reply:
Fixed, I had updated the regex without regenerating these.